app/controllers/api_controller.rb

```diff
@@ -5,6 +5,7 @@ class APIController < ActionController::API
 
   before_action :verify_token
   def verify_token
+    binding.pry
```

Comment:
I think this got added back

Reply:
I think I just missed it, I will go back and delete before merging